upgrades/steps_311.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Canonical Ltd.
```

Comment:
Maybe 2023?

Reply:
actually, i needed to delete this and the other upgrade steps